cmd/gitbase/command/server.go

```diff
@@ -193,6 +194,7 @@ func (c *Server) registerDrivers() error {
 	logrus.Debug("established connection with pilosa")
 
 	c.engine.Catalog.RegisterIndexDriver(pilosa.NewDriver(c.IndexDir, client))
+	c.engine.Catalog.RegisterIndexDriver(pilosalib.NewDriver(c.IndexDir))
```

Comment:
should we use another folder for that driver? maybe we can create a mess if we name two indexes in the same way from different drivers.

Reply:
We can do that, but in theory it should work, because it's a root folder where mapping and config file goes.  Mapping and Config are the same. The only concern I have if someone wants to create a 2 indexes for the same expression (but I think it's not possible).
So for old pilosa mapping and config goes to `root/db/table/id` with names `config.yml`, `mapping.db`. Config contains information about what driver was used to create an index.
An extra metadata for pilosalib goes under to `.pilosalib` folder under `id` folder.

So it's up to us. if we want to have 2 different roots per driver.

Reply:
I would prefer two folders to avoid unexpected problems.

Reply:
ok, fair enough